scipy/stats/tests/test_distributions.py

```diff
@@ -371,6 +371,51 @@ def test_pdf_zero(self):
         assert_equal(stats.geninvgauss.pdf(2e6, 50, 2), 0)
 
 
+class TestGenHyperbolic(object):
+    def setup_method(self):
+        np.random.seed(1234)
+
+    def test_pdf_t(self):
+        # Test Against T-Student with 1 - 100 df
+        res = []
+        for df in np.linspace(1,100,20):
+        # in principle alpha should be zero
+        # in practice for big lmbdas alpha cannot be too small else pdf pdf does not integrate
+            alpha = np.float_power(df,2)*np.finfo(np.float32).eps 
+            gh = stats.genhyperbolic(lmbda=-df/2, alpha=alpha, beta=0, delta=np.sqrt(df), mu=0)
+            x = np.linspace(gh.ppf(0.01), gh.ppf(0.99), 100)
+            res.append(np.allclose(gh.pdf(x), stats.t.pdf(x, df), atol=1e-9))
```

Comment:
No need to append these to a list. Just do:
```python3
np.testing.assert_allclose(gh.pdf(x), stats.t.pdf(x, df), atol=1e-9)
```
Please follow other examples in the file. For instance, `assert_allclose` may be imported at the top, so you might not need `np.testing`.